app/Jobs/UpdateWikiSiteStatsJob.php

```diff
@@ -26,6 +29,31 @@ public function handle (): void
         }
     }
 
+    private function updateLifecycleEvents (Wiki $wiki): void {
+        $responses = Http::pool(fn (Pool $pool) => [
+            $pool->as('revisions')->withHeaders(['host' => $wiki->getAttribute('domain')])->get(
```

Comment:
Theoretically, the value returned by this call will never change, so if we wanted to optimize for performance, we could also check if such a value has already been stored for the wiki and the only make the call if we need to.

I chose not to do this right now as we run this once a day only and performance is not an issue.